### ISSUE TITLE
QGIS crashes when try to load vectors created on Geonode as WFS

### ISSUE THREAD

**Issue by jbogdani** [GeoNode/QGISGeoNodePlugin#254]
Hello all,
I'm having some issues with loading and editing of vectors hosted in a Geonode server (v. 4.x). If I try to load a vector that has been created via Geonode, QGIS will crash and close.

To reproduce the error, please follow these simple steps:
- Create a very simple dataset in Geonode
<img width="771" alt="image" src="https://user-images.githubusercontent.com/1449293/206888875-329807e1-f56f-4383-979b-71ea9678745b.png">
- Eventually grant it editing permissions
<img width="776" alt="image" src="https://user-images.githubusercontent.com/1449293/206888918-442a41b6-2b72-460f-816a-30beb8d5561d.png">
- Open QGIS and QGIS Geonode and search for the newly created vector
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/1449293/206888996-0c9e88d8-720a-478c-bcd6-368ab6631ca4.png">
- Load it into the canvas as WFS.

If the layer us loaded as WMS no error occurs. Also no error occurs with vector files uploaded to Geonode and not created by it.

- QGIS version: 3.22.12
- QGIS Geonode 1.0.0
- MacOS Ventura 13.0.1 (22A400)

**Comment by thomschke** [comment]
same issue with:

- QGIS version: 3.30.3
- QGIS Geonode 1.0.1
- GeoNode 4.1.0
- MacOS Ventura 13.3.1 (22E772610a)

```
Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       UNKNOWN_0xD at 0x0000000000000000
Exception Codes:       0x000000000000000d, 0x0000000000000000
```

**Comment by Gpetrak** [comment]
This issue is fixed in the new version of QGISGeoNode plugin which will be available soon. It is also related with the following issue (which is also solved): #297 . So the issue will be closed.

**Comment by Gpetrak** [comment]
The PR that fixes this issue is available here: #298
